File: 03-CPP/04-condicionales/02ejercicioMayorEdad.cpp
// Ejercicio mayor de edad en C++
#include <iostream>
using namespace std;

int main()
{
    const int EDAD_ADULTO = 18;
    int edadPersona;

    // Pedirle edad al usuario
    cout << "Ingresa tu edad: " << endl;
    cin >> edadPersona;

    // Validar si es mayor de edad
    if (edadPersona >= EDAD_ADULTO)
    {
        cout << "Eres mayor de edad: " << endl;
    }
    else
    {

        cout << "Eres menor de edad: " << endl;
    }

    return 0;
}

File: 04-CSHARP/04-Condicionales/02ejercicioMayorEdad.cs
// Ejercicio mayor de edad en C#

const int EDAD_ADULTO = 18;
int edadPersona;

Console.WriteLine("Ingresa tu edad");
edadPersona = Convert.ToInt32(Console.ReadLine());

if (edadPersona >= EDAD_ADULTO)
{
    Console.WriteLine("Eres mayor de edad");
}
else if (edadPersona < 0)
{
    Console.WriteLine("Valor erroneo");
}
else
{
    Console.WriteLine("Eres menor de edad");
}

